packages/algob/src/lib/compile.ts

```diff
@@ -113,7 +116,7 @@ export class CompileOp {
 		return this.algocl.compile(code).do();
 	}
 
-	async compile(filename: string, tealCode: string, tealHash: number): Promise<ASCCache> {
+	async compile(filename: string, tealCode: string, tealHash: number, SCParams: SCParams): Promise<ASCCache> {
```

Comment:
+ Naming convention is wrong (`SCParams`). 
+ Also, shouldn't `scParams` be an optional parameter here?